spec/models/wiky_spec.rb

```diff
@@ -16,4 +16,15 @@
       expect((wiky.reload).public).to eq(true) 
     end 
   end
-end
+
+  describe ".search_for" do 
+    it "searches for words in title or body of wiki" do 
+      tomato_wiki = Wiky.create!(title: "tomato", body: "red fruit" )
+      harry_wiki = Wiky.create!(title: "Harry Potter", body: "book series")
+      expect(Wiky.search_for("tomato")).to match_array(
+        Wiky.where("title LIKE '%#{"tomato"}%' OR body LIKE '%#{"tomato"}%'"))
```

Comment:
Well done on adding test coverage for this.

It'd be a slightly more robust test if the `tomato_wiki` and `harry_wiki` objects were used instead of using `Wiky.where(...)` which, if it had a problem, may hide a bug that this test should reveal.

So for the first expectation, a more reliable test is:
```ruby
expect(Wiky.search_for("tomato")).to match_array([tomato_wiki])
```

Consider applying a similar change to lines 26&27 in this spec.